misc/ballerina-bindgen/src/main/java/org/ballerinalang/bindgen/utils/BindgenUtils.java

```diff
@@ -231,7 +301,6 @@ public static void createDirectory(String path) throws BindgenException {
     }
 
     public static Set<String> getClassNamesInJar(String jarPath) throws IOException {
```

Comment:
This method has used anywhere?

Reply:
No, ATM support for generating bindings for packages is not provided, but we intend on providing that in the future. This method could be used when implementing that feature and that is why it is not removed even though it is not used.